app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Hey there! 👋 While your project is functioning as expected, I couldn't help but notice a bit of code duplication. 🔄 Just a friendly heads-up that duplicating code goes against the principles of code quality. 🚨 Consider refactoring to enhance maintainability and prevent potential issues down the road. 🛠️ Keep up the good work! 🌟